Documentation/GeneralPrinciples/PageTree/Index.rst

```diff
@@ -124,12 +162,31 @@ page-related functions the fast way. Here is what these options do:
 - **History/Undo**: Shows the change history of the page (who did which
   changes when)
 
-- **Branch Actions**: Actions related to the whole branch
 
-  - **More options ... > Export**: Opens the export tool and preselects the selected page
+.. _new-page-drag-and-drop:
 
-  - **More options ... > Import**: Opens the import tool and preselects the selected
-    page
+Create New Pages With Drag and Drop
+===================================
+
+You can also create new pages using drag'n'drop.
+
+.. rst-class:: bignums
+
+1. Clicking on the "Create new pages" icon (top left)
```

Comment:
I'd say `top left above the page tree`, otherwise people might search in the top left of the whole backend.